src/towncrier/test/test_builder.py

```diff
@@ -8,63 +8,73 @@
 
 class TestParseNewsfragmentBasename(TestCase):
     def test_simple(self):
+        """Plain <number>.<category> should be valid with counter=0."""
```

Comment:
We can leave these comments.

Just a note, if you have time to make them better



I recommend this read ... is not log https://jml.io/test-docstrings/

Maybe
```suggestion
        """
        <number>.<category> generates a counter value of 0.
        """
```